bodhi/server/scripts/dequeue_stable.py

```diff
@@ -0,0 +1,40 @@
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""This script is responsible for moving all updates with a batched request to a stable request."""
+
+import sys
+import click
+
+from bodhi.server import config, models, Session, initialize_db
+
+
+@click.command()
+@click.version_option(message='%(version)s')
+def dequeue_stable():
+    """Convert all batched requests to stable requests."""
+    initialize_db(config.config)
+    db = Session()
+
+    try:
+        batched = db.query(models.Update).filter_by(request=models.UpdateRequest.batched).all()
+
+        for update in batched:
+            update.request = models.UpdateRequest.stable
```

Comment:
I saw in the ```models.py``` code above that it was doing something like this:

```
self.set_request(db, UpdateRequest.stable, agent)
self.request = models.UpdateRequest.stable
```

The second line of that seemed a little redundant to me since ```set_request()``` seems to do what its name implies. Anyways, I think we probably do want to call ```set_request()``` since it does do a few things other than just setting the request.

I also just noted that ```set_request()``` currently wants the db Session object which I just advised you to get rid of. This leaves you with two choices: you can refactor ```set_request()``` so that it doesn't need the db Session, or you can ignore my previous advice and we can refactor ```set_request()``` another day. I'd advise keeping the concerns separate and refactoring another day.